.github/workflows/ci_test-conda.yml

```diff
@@ -13,8 +13,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.7]
-        pytorch-version: [1.3, 1.4, 1.5, 1.6, 1.7, 1.8, 1.9]
+        python-version: ["3.7"]
+        pytorch-version: ["1.3", "1.4", "1.5", "1.6", "1.7", "1.8", "1.9", "1.10"]
```

Comment:
should remove 1.3?

Reply:
Let's first get 1.10 on board... need to find compatible TV version

Reply:
I asked because 1.4 is our minimum supported version

Reply:
yes, I know that PL min is 1.4, but till we crash to some limitations we can preserve it...